### DISCUSSION TITLE
Search on comments

### DISCUSSION THREAD

**Discussion by zikapanam** [navidrome/navidrome#2331] [Ideas]
Hello,

I'm new to navidrome ! This is a great open source soft and easy to setup ! Thx a lot.

I setup id3v2 tags in my mp3 from my rehearsal records :
- title of the song
- comment : the musicians name how are involved in the jam
- album name : the date of the rehearsal

Unfortunately, i did not find my way to add a filter on comments.

Is there a way to achieve this task ?

And what if i want to search only on title, album or comment ?

Best Regards,
Laurent

**Comment by deluan** [comment]
This is currently not possible, but will be when we implement Full Text Search (#255) or Multiple Artists (#238)
